HerPublicWebsite.BusinessLogic/QuestionnaireUpdater.cs

```diff
@@ -11,9 +11,21 @@ public Questionnaire UpdateCountry(Questionnaire questionnaire, Country country)
         return questionnaire;
     }
 
+
     public Questionnaire UpdateOwnershipStatus(Questionnaire questionnaire, OwnershipStatus ownershipStatus)
     {
         questionnaire.OwnershipStatus = ownershipStatus;
         return questionnaire;
     }
+
+    public Questionnaire UpdateAddress(Questionnaire questionnaire, OsPlacesResult addressResult)
+    {
+        questionnaire.Uprn = addressResult.Uprn;
+        questionnaire.AddressPostcode = addressResult.Postcode;
+        questionnaire.AddressLine1 = $"{addressResult.BuildingNumber} {addressResult.ThoroughFareName}";

```

Comment:
Are both of these always present in the result? If not we'll sometimes end up with an extra space

Reply:
This is also an example of something that can live in the transition from OS Places DTO object to OS Places business logic model.